aws_lambda_script/app/utils.py

```diff
@@ -0,0 +1,133 @@
+import json
+import os
+import boto3
+import jwt
+import requests
+from flask_restx import abort
+from botocore.exceptions import ClientError
+from jwt.algorithms import RSAAlgorithm
+
+# Connecting to S3
+BUCKET_NAME = "keh-tech-audit-tool"
```

Comment:
I _think_ bucket names have to be unique across envs so this will probably need to be an env variable

Reply:
Changed to os.getenv("TECH_AUDIT_DATA_BUCKET")